src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -27,6 +27,10 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
 
 .bcs-Task-statusContainer {
     margin-top: 2 * $bcs-task-grid;
+
+    .bcs-TaskCompletionRuleIcon {
```

Comment:
I think if you are using this classname it can live in a file imported by the icon itself. If it is passed in from the parent `.bcs-Task-completionRule` would probably be the preferred way of using the naming convention. In both cases it doesn't need nesting; that would create unnecessary specificity.